src/queue/multilevel.rs

```diff
@@ -26,20 +28,23 @@ const CHANCE_RATIO: u32 = 4;
 
 const DEFAULT_CLEANUP_OLD_MAP_INTERVAL: Duration = Duration::from_secs(10);
 
-/// When total elapsed time exceeds this value, it will try to adjust level
-/// chances and reset the total elapsed time.
-const ADJUST_CHANCE_INTERVAL: Duration = Duration::from_secs(1);
+/// When local total elapsed time exceeds this value in microseconds, the local
+/// metrics is flushed to the global atomic metrics and try to trigger chance
+/// adjustment.
+const FLUSH_LOCAL_THRESHOLD_US: i64 = 100_000;
+
+/// When the incremental total elapsed time exceeds this value, it will try to
+/// adjust level chances and reset the total elapsed time.
+const ADJUST_CHANCE_INTERVAL_US: i64 = 1_000_000;
 
 /// When the deviation between the target and the actual level 0 proportion
 /// exceeds this value, level chances need to be adjusted.
 const ADJUST_CHANCE_THRESHOLD: f64 = 0.05;
 
-// The chance values below are the numerators of fractions with u32::max_value()
-// as the denominator.
-const INIT_LEVEL0_CHANCE: u32 = 3_435_973_836; // 0.8
-const MIN_LEVEL0_CHANCE: u32 = 1 << 31; // 0.5
-const MAX_LEVEL0_CHANCE: u32 = 4_209_067_949; // 0.98
-const ADJUST_AMOUNT: u32 = (MAX_LEVEL0_CHANCE - MIN_LEVEL0_CHANCE) / 8; // 0.06
+const INIT_LEVEL0_CHANCE: f64 = 0.8;
```

Comment:
Could we add some description about this value?

Reply:
I add some comments to it. PTAL